cmd/markdown/atmos_terraform_plan_usage.md

```diff
@@ -0,0 +1,8 @@
+Examples:
```

Comment:
"Examples" should probably be defined in the template, so it's the same style as the rest of the sections in `--help`

Reply:
This was intermediate. This is fixed now